tools/shoestring/tests/wizard/test_setup_file_generator.py

```diff
@@ -173,8 +174,9 @@ async def _assert_can_prepare_shoestring_files(expected_node_features, node_type
 					None,
 					None),
 				'voting': VotingScreen(kwargs.get('is_voting_active', False)),
-				'node-settings': NodeSettingsScreen('san.symbol.ninja', 'Symbol San', kwargs.get('api_https', False), None)
-			}, Path(output_directory))
+				'node-settings': NodeSettingsScreen('san.symbol.ninja', 'Symbol San', kwargs.get('api_https', False), None),
+				'bootstrap': BootstrapScreen(False, None, False),
```

Comment:
Do you think it makes sense to add `test_can_prepare_shoestring_files_bootstrap_import_active` in test?

Reply:
What will this test do?  When the bootstrap import is active, it dispatches the ``import-bootstrap`` command.